cmd/minishift/cmd/hostfolder/add.go

```diff
@@ -20,45 +20,223 @@ import (
 	"fmt"
 	"runtime"
 
-	hostfolderActions "github.com/minishift/minishift/pkg/minishift/hostfolder"
+	"errors"
+	miniConfig "github.com/minishift/minishift/pkg/minishift/config"
+	hf "github.com/minishift/minishift/pkg/minishift/hostfolder"
```

Comment:
Not a huge fan of abbrev. when concerning vars or libs.

Reply:
+1

Reply:
I renamed it to _hostFolderConfig_.